Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -13,27 +14,33 @@
 
 use Doctrine\Common\Annotations\AnnotationReader;
 use Doctrine\Common\Annotations\PhpParser;
+use Doctrine\Common\Annotations\Reader;
 use Doctrine\ORM\Mapping as ORM;
 use Doctrine\Persistence\Proxy as DoctrineProxy;
-use Neos\Flow\Annotations as Flow;
-use Neos\Cache\Frontend\FrontendInterface;
+use InvalidArgumentException;
```

Comment:
i think you configured your ide like this...

i dont use it myself and it is an anomaly to the rest of neos/flow
just wanted to start the diskussion.

do we want this for new code?

Reply:
Hmm, I don't know … I like it and do that for a couple of years now in my own code. It's the easiest rule: Import if possible.

Reply:
Good point. I dislike this and exclude the global namespace from being imported in PhpStorm.

For me imports like these do not add anything useful.

Reply:
slack has your back @kdambekalns https://neos-project.slack.com/archives/C04Q0TC15/p1683558099679779 😎